Gems/PhysX/Code/Tests/PhysXColliderComponentModeTests.cpp

```diff
@@ -64,6 +64,26 @@ namespace UnitTest
         }
     };
 
+    void DragMouse(
```

Comment:
+1 to the helper function 👍 (maybe in a future PR we can move this to `AzToolsFramework` so other tests can use it outside `PhysX`)

Reply:
Yeah, that would be a good idea since there's one in LmbrCentral as well.

Reply:
Moved now to AzManipulatorTestFrameworkUtils

Reply:
Fantastic thanks!